site2/docs/deploy-bare-metal.md

```diff
@@ -447,13 +445,13 @@ bin/pulsar-admin functions create \
   --name exclamation
 ```
 
-Check if the function runs as expected by [triggering](functions-deploying.md#triggering-pulsar-functions) the function.
+Check whether the function runs as expected by [triggering](functions-deploying.md#triggering-pulsar-functions) the function.
 
 ```bash
 bin/pulsar-admin functions trigger --name exclamation --trigger-value "hello world"
 ```
 
-You can see the output as below:
+You should see the following output:
```

Comment:
"can" indicates the correct meaning.

Reply:
'Can' implies the ability to do something. 'Should' is used to indicate what is probable or what is intended. When a person runs the command, they should see something, but if something was incorrect they would not. That is the intent I was conveying.